docs/src/examples/Tree.jl

```diff
@@ -0,0 +1,73 @@
+# # Tree
+#
+#md # [![Binder](https://mybinder.org/badge_logo.svg)](@__BINDER_ROOT_URL__/generated/Tree.ipynb)
+#md # [![nbviewer](https://img.shields.io/badge/show-nbviewer-579ACA.svg)](@__NBVIEWER_ROOT_URL__/generated/Tree.ipynb)
+#
+# In this file, we present the Tree data structure, which is a tree composed of costs on edges and an underlying metric between node states.
+# In this simple example, the states are two-dimensional ellipsoids.
+# 
+# First, let us import a few packages that are necessary to run this example.
+using LinearAlgebra, Plots, Colors
+
+# The main package [Dionysos](https://github.com/dionysos-dev/Dionysos.jl) provides most important data structures that we will need.
+using Dionysos
+UT = Dionysos.Utils
+
+# We define the underlying metric between node states
+function distance(E1, E2)
```

Comment:
This could be defined at the same level as UT.pointCenterDistance in a easier way like
```
distance(E1::Ellipsoid, E2::Ellipsoid) = distance_function_wathever(E1.c, E2.c)
```